### ISSUE TITLE
arangosh --javascript.execute-string does not work as expected

### ISSUE THREAD

**Issue by leowmjw** [arangodb/arangodb#1809]
On OSX, arangosh --javascript.execute-string does not work as expected.

**leow$ arangosh --javascript.execute-string "db._useDatabase('sinar'); db.par.truncate();" --log.level debug**

The collection specified is not truncated as would be expected.

The same command executed on a Linux box works fine.  This might be another case of CLI working as expected on Linux but not on OSX ArangoDB.app.

When the same statement is placed inside a JS file and called via --javascript.execute; things work as expected.

ArangoDB 2.8.6

**Comment by dothebart** [comment]
ok, so this is most probably related to #1779 ; can you revalidate by removing the blank or putting in a require("internal").print() call?

**Comment by leowmjw** [comment]
@dothebart Confirmed; execution of statement stops on the first blank space it encounters.

Does NOT Work (WITH Spaces):
```
    echo "In setup_aduanku_aduanku_collections .."
    arangosh --javascript.execute-string \
        "db._useDatabase('sinar'); db._createDocumentCollection('aduanku'); " \
        --log.level debug
```

Works (NO Spaces):
```
    echo "In setup_aduanku_aduanku_collections .."
    arangosh --javascript.execute-string \
        "db._useDatabase('sinar');db._createDocumentCollection('aduanku'); " \
        --log.level debug
```

**Comment by jsteemann** [comment]
Same root cause as #1779.

The issue is caused by the parameters to the executables in the Mac app version being wrapped in an extra shell script, which passes parameters with $* instead of $@.
This will be fixed in the next 2.8 release.

**Comment by dothebart** [comment]
This was also fixed with 2.8.9
